js/src/ui/Loading/loading.js

```diff
@@ -21,15 +21,22 @@ import styles from './loading.css';
 
 export default class Loading extends Component {
   static propTypes = {
+    className: PropTypes.string,
     size: PropTypes.number
   };
 
+  static defaultProps = {
+    className: '',
+    size: 2
+  };
+
   render () {
-    const size = (this.props.size || 2) * 60;
+    const { className, size } = this.props;
+    const _size = size * 60;
```

Comment:
Prefer to keep `_name` to function params where supplied. So may make more sense as `calculatedSize`.